modules/python3/bindings/src/pyglmtypes.cpp

```diff
@@ -76,19 +77,14 @@ void addInit(py::classh<V>& pyv) {
 }  // namespace
 
 template <typename T, int Dim>
-void vecx(py::module& m, const std::string& prefix, const std::string& name,
-          const std::string& postfix) {
+void vecx(py::module& m, std::string_view prefix, std::string_view name, std::string_view postfix) {
```

Comment:
putting these internal function in a anonymous namespace is probably a good idea.